bin/alluxio

```diff
@@ -257,13 +257,8 @@ function main {
     runJavaClass "$@"
   ;;
   "runTest")
+    PARAMETER="configured" # instructs runTests to use the configured master address
```

Comment:
to be consistent with other places, would it make sense to omit the value altogether ... so if `args[0]` is empty we will default to using the configuration